cfy_cluster_manager/main.py

```diff
@@ -598,17 +598,32 @@ def _validate_config_paths(existing_vms_list, using_three_nodes, errors_list):
                            'all instances or none of them.')
 
 
+def _validate_vms_not_deuplicated(existing_vms_dict, errors_list):
+    existing_vms_ips = set()
+    for vm_name, vm_dict in existing_vms_dict.items():
+        vm_private_ip = vm_dict.get('private_ip')
+        if not vm_private_ip:
+            errors_list.append(
+                'private_ip should be provided for {0}'.format(vm_name))
+            continue
+        for vm in existing_vms_ips:
+            if vm_private_ip == vm[1]:
+                errors_list.append(
+                    'The private_ip of {0} and {1} is the same'.format(
```

Comment:
I feel like this should say something more- perhaps a call to action: "The private_ips of {0} and {1} are the same. Please ensure all provided VMs have different IPs." (or something more elegant)

Reply:
`private_ip of all nodes must be different, but {0} and {1} have {2}` is my wording

Reply:
Right! Rephrased it. I'll remember this comment about adding a call to action.